recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
 {% set name = "ydata-profiling" %}
-{% set version = "4.1.1" %}
+{% set version = "4.2.0" %}
```

Comment:
Could you please update to 4.3.1 https://github.com/ydataai/ydata-profiling/releases/tag/v4.3.1 ?

Reply:
sure